lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -1595,6 +1592,10 @@ bool PrimUpdaterManager::removePullParent(const MDagPath& parentDagPath)
         return false;
     }
 
+    Ufe::Path pulledPath;
+    TF_VERIFY(readPullInformation(parentDagPath, pulledPath));
+    TF_VERIFY(_pulledPrims.remove(pulledPath) != nullptr);
```

Comment:
When we remove a pull parent, we remove the corresponding variant set configuration in the Trie.

Reply:
IDK if it would be best to remove it at the end when successful or not. OTOH, half-made push will always be problematic.